### ISSUE TITLE
Institutions return exit action on sandbox

### ISSUE THREAD

**Issue by roktoshl** [plaid/link#180]
|       Name       | Value |
|------------------|-------|
| Link `env`       |   sandbox    |
| Browser          |   webview    |
| Operating system |   Android    |

### A detailed description of the steps to reproduce the issue

* Got an error when openening AmericanExpress/PayPal/...
* Error: plaidlink://exit?request_id&status=requires_credentials
* Links are opened by adding an "insitution" parameter (?institution=ins_5) as such https://cdn.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&clientName={client_name}&selectAccount=false&key={key}&webhook={webhook}&apiVersion=v2&env=sandbox&institution=ins_5&product=auth

### What you expected to see?

Login screen.

### What you saw instead?

Blank screen with the error mentioned above.

### A screenshot that shows the state of the UI (if applicable)

/

**Comment by michaelckelly** [comment]
@roktoshl - Thanks for opening this issue up. Are you having trouble integrating Link into your Android app or is this an issue with a user attempting to link a particular bank.

In a Webview, Link uses a custom URL scheme `plaidlink://` to communicate information between Link and your app. In this case, the user exited Link -- typically, your application code will handle this case. See the [example app](https://github.com/plaid/link/blob/master/webviews/examples/android/app/src/main/java/plaid/samplewebviewapp/MainActivity.java#L74-L87) for a code snippet.

**Comment by roktoshl** [comment]
@michaelckelly The problem is that on some banks the login screen opens up normally but some bank react differently when opening the login screen. I had no trouble integrating Link webview into Android however.

**Comment by roktoshl** [comment]
@michaelckelly Any news about this one?

**Comment by michaelckelly** [comment]
@roktoshl Are you using Link's custom initializers -- so skipping the 'Institution Select' view and jumping straight to the credentials view for a particular institution? If so, the issue may be the institution ID that you're using. All of the institution IDs in our current API are in the form `ins_`, i.e. `ins_10` is American Express. For a list of the most popular institution IDs, check our [Institution API docs](https://plaid.com/docs/api/#institutions). 

In your case, try `ins_10` for American Express and `ins_22` for PayPal.

Let me know if this helps!

**Comment by roktoshl** [comment]
@michaelckelly Yes exactly, we jump straight to the credentials view. I am passing the same institution ids for PayPal - ins_22 and for American Express ins_10, I still get the exit action however. Can you check if this is an issue only on sandbox, because we are currently testing this environment and haven't switched to production yet?

**Comment by roktoshl** [comment]
@michaelckelly Any news on this?
